lib/zoom/actions.rb

```diff
@@ -40,10 +40,15 @@ def self.make_request(client, method, parsed_path, params, base_uri)
           params = params.require(path_keys) unless path_keys.empty?
           params_without_required = required.empty? ? params : params.require(required)
           params_without_required.permit(permitted) unless permitted.empty?
-          response = Zoom::Actions.make_request(self, method, parsed_path, params, base_uri)
+          response = Zoom::Actions.make_request(self, method, parsed_path, params, base_uri,
+                                                url_encoded: Zoom::Actions.form_url_encoded_actions.include?(name))
           Utils.parse_response(response)
         end
       end
     end
+
+    def self.form_url_encoded_actions
+      %w[access_tokens refresh_tokens revoke_tokens]
```

Comment:
It seems odd that this generic meta-programming class which provides a DSL for defining actions also would know which specific actions require form URL encoding. This seems like something that the `lib/zoom/actions/token.rb` class should know about.

Reply:
I think @gerryster is right. Having this method in the token.rb makes more sense.